packages/x64dbgpy.vm/x64dbgpy.vm.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>x64dbgpy.vm</id>
-    <version>1.0.56.20211021</version>
+    <version>1.0.56.20230405</version>
```

Comment:
Hasn't the tool version changed? The version should be the tool version. In case the tool version uses the forth segment we replace it by the date. See https://github.com/mandiant/VM-Packages/wiki/Package-Structure#version